### DISCUSSION TITLE
Charger recommender on Tesla Android

### DISCUSSION THREAD

**Discussion by hawajka76** [tesla-android/issue-tracker#283] [General]
After having some 'interesting' charging experiences especially at none Tesla chargers I've made an app with a charger recommender, amenity and safety based scores. 
I don't have a Tesla myself and was wondering if any of you would be interested in testing if the app runs on Tesla Android? 
It would need to use the GPS to display the user's position and for calculating the journey time for the charger recommender. It features both Tesla and none Tesla charge points. 
You can download it via the Play store or our website. www.electro-app.com.

**Comment by josbmw** [comment]
I had a GPS receiver in combination with Teslaandroid to run Waze but returned it since it did not work with ABRP. It's waiting for the new release which incorporates GPS data from the Tesla browser I think...

**Reply by mikegapinski** [reply]
It spent two last weeks in the beta circle, It should be good to go soon

**Reply by hawajka76** [reply]
I see, if Teslaandroid incorporates GPS data from the Tesla browser it should work then I guess? 
Was also curious if a regular app will work or if Android Auto compatibility is required?

**Reply by mikegapinski** [reply]
Yup 
<img width="1567" alt="image" src="https://github.com/tesla-android/issue-tracker/assets/930055/d0990176-b3cf-4578-8685-51244b003f65">

**Reply by hawajka76** [reply]
Nice that is awsome! Was wondering how to bring it to Tesla drivers without doing a web version.
